new-backoffice/extending/context-api/README.md

```diff
@@ -40,20 +40,93 @@ If you need to consume a Context API from a non-controller host, then look at th
 A Context Token is generally a string matched with a type. In this way, users of the token can be sure to get the right type of context.
 
 ```ts
-import { ContextToken } from '@umbraco-cms/backoffice/context';
+import { UmbContextToken } from "@umbraco-cms/backoffice/context";
 
 type MyContext = {
-	foo: string;
-	bar: number;
+    foo: string;
+    bar: number;
 };
 
-const MY_CONTEXT_TOKEN = new ContextToken<MyContext>('My.Context.Token');
+const MY_API_TOKEN = new UmbContextToken<MyContext>("My.Context.Token");
 ```
 
+### **Context Token with an API Alias**
+
+For additions to Contexts, we can use the API Aliases to identify the additional API. Using the same Context Alias for additional APIs will ensure that such API has to be present with the first encounter of that Context Alias. Otherwise, a request will be rejected.
+In other words, if the addition is not part of the nearest matching Context, then a request will be rejected.
+
+{% hint style="info" %}
+Using API Alias only provides value when two or more APIs should share the same Context.
+This is needed for Context Extensions that are provided along with other Contexts.
+{% endhint %}
+
+```ts
+import { UmbContextToken } from "@umbraco-cms/backoffice/context";
+
+type MyAdditionalContext = {
+    additional: string;
+};
+
+const MY_ADDITIONAL_API_TOKEN = new UmbContextToken<MyAdditionalContext>(
+    "My.ContextFrame.Alias",
+    "My.API.Alias"
+);
+```
+
+The Token declared above can then be used to provide an additional Context API at the same Element as another Context API is provided at.
+The example below shows how the two APIs are made available.
+
+```ts
+const contextElement = new UmbLitElement();
+contextElement.provideContext(
+    MY_API_TOKEN,
+    new MyAPiFromSomewhereNotPartOfThisExample()
+);
+contextElement.provideContext(
+    MY_ADDITIONAL_API_TOKEN,
+    new MyAdditionalAPiFromSomewhereNotPartOfThisExample()
+);
+
+const consumerElement = new UmbLitElement();
+contextElement.appendChild(consumerElement);
+consumerElement.consumeContext(MY_API_TOKEN, (context) => {
+    console.log("I've got the default api", context);
+});
+consumerElement.consumeContext(MY_ADDITIONAL_API_TOKEN, (context) => {
+    console.log("I've got the additional api", context);
+});
+```
+
+This is no different from using two different Context Aliases, the difference first comes into play once one of these is not provided or provided on a parent element as displayed in the example below:
```

Comment:
to keep the sentences shorter than 25 words as per our styleguide:
```suggestion
This is no different from using two different Context Aliases. The difference first comes into play once one of these is not provided or on a parent element as displayed in the example below:
```

Reply:
Sounds good! I have only made the suggestions as the word "provided" was twice in same sentence 🙈 👍